src/app/(pages)/home/_components/SponsorsBlock.tsx

```diff
@@ -0,0 +1,12 @@
+import BlockContainer from '@/components/BlockContainer';
+import SponsorButton from './SponsorButton';
+import ServerCard from './ServerCard';
+
+export default function SponsorsBlock() {
+  return (
+    <BlockContainer title="Our Sponsors" padding="less" inner margin="bottom">
+      <ServerCard />
```

Comment:
Our Sponsor cards need to show up as a carousel in mobile view. Let's implement this after moving the sponsors object and the sponsors.map() call over to this file.